gvm

```diff
@@ -226,7 +233,17 @@ update() {
         EXPECTEDSUM=$(curl -sL "$SUMPATH")
     
         # shellcheck disable=SC2086
-        FILESUM=$(cat $0 | shasum -a 256)
+
+          if [[ "$OSTYPE" == "msys"* ]] 
+        then
+            FILESUM=$(certutil -hashfile $0 sha256 | tail -n2 | head -n1)
+            EXPECTEDSUM=$(curl -sL "$SUMPATH" | sed 's/  -//')  # ⚠️ additional 2xspace + dash may also need to be addressed on unix installs
```

Comment:
This was interesting as I noticed the additional spaces and dash being appended to $EXPECTEDSUM coming from the $SUMPATH request.  I don't have a lot of experience in bash so this may be expected in the Unix environment.  Specifically when using shasum.   Just thought I'd call it out that I had to strip these appended characters in order to accurately compare the file downloaded to the expected checksum.